src/components/Calculator.js

```diff
@@ -1,39 +1,84 @@
+    } else if (operators.includes(symbol)) {
+      this.setState({
+        operation: symbol,
+        next: total,
+        total: null,
+        display: `${display} ${symbol} `,
+      });
+    } else if (symbol === '=') {
+      this.setState(calculate({ total, next, operation }, symbol));
+      this.setState({ display: total });
+    } else {
+      this.setState(calculate({ total, next, operation }, symbol));
+      this.setState({ display: total });
+    }
+    console.log(this.state);
+  }
+
   render() {
+    const { display } = this.state;
     return (
       <div>
-        <div className="screen">0</div>
+        <div className="screen">{display}</div>
         <div className="row">
-          <Key symbol="AC" />
-          <Key symbol="+/-" />
-          <Key symbol="%" />
-          <Key symbol="/" color="orange" />
+          <Key symbol="AC" retrieveSymbol={this.retrieveSymbol} />
+          <Key symbol="+/-" retrieveSymbol={this.retrieveSymbol} />
+          <Key symbol="%" retrieveSymbol={this.retrieveSymbol} />
+          <Key symbol="÷" color="orange" retrieveSymbol={this.retrieveSymbol} />
         </div>
         <div className="row">
-          <Key symbol="7" />
-          <Key symbol="8" />
-          <Key symbol="9" />
-          <Key symbol="X" color="orange" />
+          <Key symbol="7" retrieveSymbol={this.retrieveSymbol} />
+          <Key symbol="8" retrieveSymbol={this.retrieveSymbol} />
+          <Key symbol="9" retrieveSymbol={this.retrieveSymbol} />
+          <Key symbol="x" color="orange" retrieveSymbol={this.retrieveSymbol} />
         </div>
         <div className="row">
-          <Key symbol="4" />
-          <Key symbol="5" />
-          <Key symbol="6" />
-          <Key symbol="-" color="orange" />
+          <Key symbol="4" retrieveSymbol={this.retrieveSymbol} />
+          <Key symbol="5" retrieveSymbol={this.retrieveSymbol} />
+          <Key symbol="6" retrieveSymbol={this.retrieveSymbol} />
+          <Key symbol="-" color="orange" retrieveSymbol={this.retrieveSymbol} />
         </div>
         <div className="row">
-          <Key symbol="1" />
-          <Key symbol="2" />
-          <Key symbol="3" />
-          <Key symbol="+" color="orange" />
+          <Key symbol="1" retrieveSymbol={this.retrieveSymbol} />
+          <Key symbol="2" retrieveSymbol={this.retrieveSymbol} />
+          <Key symbol="3" retrieveSymbol={this.retrieveSymbol} />
+          <Key symbol="+" color="orange" retrieveSymbol={this.retrieveSymbol} />
         </div>
         <div className="row">
-          <Key symbol="0" color="double" />
-          <Key symbol="." />
-          <Key symbol="=" color="orange" />
+          <Key symbol="0" color="double" retrieveSymbol={this.retrieveSymbol} />
+          <Key symbol="." retrieveSymbol={this.retrieveSymbol} />
+          <Key symbol="=" color="orange" retrieveSymbol={this.retrieveSymbol} />
```

Comment:
- [OPTIONAL] I think you might need to a re-arrange your  UI. The calculator design is all the way to the top left of the screen.

See screenshot|
:---------------------------:|

![a1](https://user-images.githubusercontent.com/39523312/158731631-2446a879-6ced-4290-9488-52ad06fdfda8.png)